admin/settings/plugins.php

```diff
@@ -416,6 +416,13 @@
     }
 }
 
+// Column sort order.
+if ($hassiteconfig) {
+    $url = new moodle_url("/question/qbanksettings.php", ['section' => 'columnsortorder']);
+    $temp = new admin_externalpage('settingqbank', get_string('columnsortorder', 'question'), $url);
+    $ADMIN->add('qbanksettings', $temp);
+}
```

Comment:
We should change, please see the main comment in the review.

Reply:
Changed